src/wutil/mod.rs

```diff
@@ -537,10 +537,16 @@ impl FileId {
         result.change_seconds = buf.st_ctime;
         result.mod_seconds = buf.st_mtime;
         #[allow(clippy::unnecessary_cast)] // platform-dependent
+        #[cfg(not(target_os = "netbsd"))]
         {
             result.change_nanoseconds = buf.st_ctime_nsec as _;
             result.mod_nanoseconds = buf.st_mtime_nsec as _;
         }
+        #[cfg(target_os = "netbsd")]
+        {
+            result.change_nanoseconds = buf.st_ctimensec as _;
```

Comment:
> Especially the name difference is one I just find fudging annoying. I had hoped we could get away from pointless stuff like that.

I think the `libc` crate already does a bunch of platform-dependent renaming of struct fields, so this can be fixed over there eventually.